### ISSUE TITLE
QA Ph3_Accessibility: My Schedule - Settings link not accessible with keyboard only 

### ISSUE THREAD

**Issue by brittasinnott** [GoogleChrome/ioweb2016#629]
Using the keyboard only to navigate, the settings link on the My Schedule page (that appears when the message regarding notifications has not yet been closed) cannot be accessed.  

Steps: 
1. Go to https://io-webapp-staging.appspot.com/io2016/
2. Sign in
3. Select Schedule 
4. Select the My Schedule tab 
5. Use the keyboard to navigate to the Settings link on the message regarding notifications 
6. Once landed on Settings tap enter
7. Note there is no response 

Chrome 49 Mac OS X  10.11 

![gio_629](https://cloud.githubusercontent.com/assets/15128124/14443314/454094b0-fff3-11e5-891b-f67a2543d6c1.png)
